composer.json

```diff
@@ -29,25 +29,25 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^8.0",
-        "phpunit/phpunit": "^9.5",
+        "php": "8.1 - 8.4",
+        "phpunit/phpunit": "^10.5.45 || ^11.5 || ^12.1",
```

Comment:
Use one version

Reply:
Can I use phpunit 12.1?
I want to use the latest version of phpunit in my application

Reply:
> Can I use phpunit 12.1? I want to use the latest version of phpunit in my application

Use version that support PHP 8.1. Seems, this is 10.

Reply:
Oh. This is not development dependency, so it's OK to use several versions of PHPUnit.